docs/features.md

```diff
@@ -209,10 +209,69 @@ Make Julia start listening on a given port and return lint messages to requests
 This feature is useful when you want to lint julia code in a non julia environment.
 
 Existing plugins:
+
  * Sublime Text: [SublimeLinter-contrib-julialintserver](https://github.com/invenia/SublimeLinter-contrib-julialintserver)
  * linter-julia for Atom: [linter-julia](https://github.com/TeroFrondelius/linter-julia)
 
-The protocol for the server is:
+The new protocol for the server is JSON in both input and output:
+```json
+{
+  'file': "path_to_the_file",
```

Comment:
I guess this is CSON actually. I will call stringify and change it here. [FYI the actual code from linter-julia](https://github.com/TeroFrondelius/linter-julia/blob/db8daf97f2cc52676597033707f8c6525d55e810/lib/linter-julia.coffee#L78-L86).